lib/src/dynamics/fixture.dart

```diff
@@ -11,9 +11,10 @@ import '../settings.dart' as settings;
 class Fixture {
   double _density = 0.0;
 
+  // TODO.0llie null-safety breaking change: Fixture should need a body?
```

Comment:
Yes, I think that is a good thing to force, a fixture is not much use without a body, also since fixtures should be created from the body with `body.createFixture`